packages/device-utils/src/firmwareUtils.ts

```diff
@@ -18,9 +18,15 @@ export const getFirmwareVersion = (device?: PartialDevice) => {
     return `${features.major_version}.${features.minor_version}.${features.patch_version}`;
 };
 
+export const isBitcoinOnlyFirmware = (device?: PartialDevice) =>
```

Comment:
nit: since a device is accepted and not the FW – `hasBitcoinOnlyFirmware`

Reply:
89150d1812f8e4ca7442f7d51eb90d62337066a3